clang/test/3C/itypes_for_extern.c

```diff
@@ -30,14 +30,19 @@ int *baz(int *a, int len, int *b) {
   return a;
 }
 
+// REVIEW: The problem described in the following comment has been fixed roughly
+// in the "first way": the canonical form of the type from mkString no longer
+// has the space, so the test will pass as long as the original code didn't have
```

Comment:
The remaining issue can be demonstrated by a simpler example: 
```c
void test(int *a : itype(_Ptr<int>)) {}
```
When converted with `3c -itypes-for-extern`, the function is rewritten even though it hasn't changed.

Reply:
Opened #715

Reply:
Thanks for filing an issue.  I assume I should just remove this comment now.